### PR TITLE
 Add Multipoint support #189 

### PR DESCRIPTION
**Pull request by frodrigo**

### CODE REVIEW

**Review by wazolab — APPROVED** [review]

File: components/Map/MapBase.vue [hunk 1]
```diff
@@ -173,10 +173,34 @@ export default defineNuxtComponent({
...
+          } as ApiPoi))
+        }
+        else {
+          return [feature]
+        }
+      })
+        .flat()
+        .filter(feature => !!feature)
+        .map((feature, index) => {
+          feature.id = index
```
Comment (wazolab) [inline components/Map/MapBase.vue:199]: Maybe you need to add this field in test case as well ?

File: components/MainMap/MapFeatures.vue [hunk 2]
```diff
@@ -373,12 +374,18 @@ export default defineNuxtComponent({
...
+        type: 'Polygon',
+        coordinates: [[
+          [mapBound[0][0], mapBound[0][1]],
+          [mapBound[1][0], mapBound[0][1]],
+          [mapBound[1][0], mapBound[1][1]],
+          [mapBound[0][0], mapBound[1][1]],
+          [mapBound[0][0], mapBound[0][1]],
+        ]],
+      } as Polygon
+      const isOneInView = this.features.some(feature => booleanIntersects(bounds, feature.geometry as Exclude<GeoJSON.Geometry, GeoJSON.GeometryCollection>))
```
[the remark this answers is not in the document]
Comment (wazolab) [reply components/MainMap/MapFeatures.vue:388]: tep sorry didn't see it ...

**Review by wazolab — APPROVED** [review]

File: components/Map/MapBase.vue [hunk 3]
```diff
@@ -220,7 +244,7 @@ export default defineNuxtComponent({
         tolerance: 0.6,
         data: {
           type: 'FeatureCollection',
-          features,
+          features: this.featuresPrepare(this.features),
```
Comment (wazolab) [inline components/Map/MapBase.vue:247]: Why not using `features` from `initPoiLayer` args ? instead of `this.features` ?

Comment (frodrigo) [reply components/Map/MapBase.vue:247]: Yes, looks like it need to be fixed.

File: cypress/e2e/tracking.cy.ts [hunk 4]
```diff
@@ -23,11 +23,11 @@ describe('home content', () => {
     localStorage.setItem('cookie:accepted', 'true')
     mockSSRAPI(hostnames, teritorioReferenceAPIFixture)
     cy.intercept(
-      '/content/api.teritorio/geodata/v0.1/dev/tourism/pois/category/22.geojson?geometry_as=point&short_description=true',
+      '/content/api.teritorio/geodata/v0.1/dev/tourism/pois/category/22.geojson?geometry_as=point&short_description=false',
```
Comment (wazolab) [inline cypress/e2e/tracking.cy.ts:26]: We will have to set it to `true` again on develop since #236

File: cypress/e2e/tracking.cy.ts [hunk 5]
```diff
@@ -23,11 +23,11 @@ describe('home content', () => {
...
       { body: poisCategory22 },
     )
     cy.intercept(
-      '/content/api.teritorio/geodata/v0.1/dev/tourism/pois/category/211.geojson?geometry_as=point&short_description=true',
+      '/content/api.teritorio/geodata/v0.1/dev/tourism/pois/category/211.geojson?geometry_as=point&short_description=false',
```
Comment (wazolab) [inline cypress/e2e/tracking.cy.ts:30]: We will have to set it to `true` again on develop since #236

File: components/Map/MapPois.vue [hunk 6]
```diff
@@ -75,7 +75,7 @@ export default defineNuxtComponent({
       if (
```
Comment (frodrigo) [inline components/Map/MapPois.vue:75]: It think the whole function is wrong.

Should just be, without if ?
```js
getBBoxFeatures(
            this.features.filter(feature => feature.geometry),
          ) || this.defaultBounds
```

**Comment by frodrigo** [comment]
Let's merge it!

**Pull request merged**